packages/blocks/src/components/drag-handle.ts

```diff
@@ -474,6 +482,104 @@ export class DragHandle extends WithDisposable(LitElement) {
     );
   }
 
+  static clacTarget(
```

Comment:
Is this a typo of `calcTarget`? Also, having the return value of this function typed will look better, current implementation looks a bit ad-hoc.